packages/front-end/pages/datasources/queries/[did].tsx

```diff
@@ -86,6 +92,8 @@ const DataSourceQueries = (): React.ReactElement => {
     );
   }
 
+  const supportsQueryCancellation = ["athena", "bigquery"].includes(d.type);
```

Comment:
Is there somewhere better that this can live? The source of truth is really whether the backend integration includes a `cancelQuery` function so that can't exactly be checked from the frontend code here